presto-main/src/main/java/com/facebook/presto/server/RequestErrorTracker.java

```diff
@@ -44,38 +47,64 @@
 import static com.facebook.presto.util.Failures.WORKER_NODE_ERROR;
 import static java.lang.String.format;
 import static java.util.Objects.requireNonNull;
+import static java.util.concurrent.TimeUnit.MILLISECONDS;
 import static java.util.concurrent.TimeUnit.NANOSECONDS;
 import static java.util.concurrent.TimeUnit.SECONDS;
 
 @ThreadSafe
 public class RequestErrorTracker
 {
     private static final Logger log = Logger.get(RequestErrorTracker.class);
-
+    private static final List<Duration> LEAF_BACKOFF_DELAY_INTERVALS = ImmutableList.<Duration>builder()
```

Comment:
add a comment to explain why a different backoff interval is needed for leaf nodes.